src/tools/sss_override.c

```diff
@@ -333,13 +333,21 @@ static struct sysdb_attrs *build_attrs(TALLOC_CTX *mem_ctx,
         if (ret != EOK) {
             goto done;
         }
+    }else {
+        fprintf(stderr, ("Setting uid=0 is not allowed\n"));
```

Comment:
I'm guessing here we want this message to be exposed to the user. So, it would be nice if it's also translated.
If that's the case, you should do something like:
`fprintf(stderr, _("Setting uid as 0 is not allowed\n"));`

If that's not the case, DEBUG macro would be more appropriate.